### DISCUSSION TITLE
md:w-[390px] works but not md:w-[537px]

### DISCUSSION THREAD

**Discussion by mongchanghsi** [tailwindlabs/tailwindcss#13851] [Help]
Very weird question but somehow when I am applying md:w-[390px] it appear in both the html class and the styles log. But md:w-[573px] it appear in html class but it does not appear in the styles log. For some weird reason it is not picking up other value other than 390px. Any idea what could be the possible issue or where can I start debugging this?

I have checked in like global config whether is there a fixed 390px being applied but nothing is configured.

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 

Otherwise, consider providing a project that reproduces the unexpected behavior.
